cmake/configs/aarch64/linux/config.cmake

```diff
@@ -13,7 +13,11 @@ set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_GNU_SOURCE" CACHE STRING "" FORCE)
 set(WT_BUFFER_ALIGNMENT_DEFAULT "4096" CACHE STRING "")
 
 # ARMv8-A is the 64-bit ARM architecture, turn on the optional CRC instructions.
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=armv8-a+crc" CACHE STRING "" FORCE)
+if (HAVE_NEOVERSE)
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mcpu=neoverse-v1" CACHE STRING "" FORCE)
```

Comment:
I think it is more appropriate and consistent to target the first ARM ISA that first supports the instructions: `-march=armv8.2-a` than a specific CPU

Reply:
Interestingly this fails to compile with `armv8.2-a` but will compile under `armv8.3-a`. gcc produces:
```
/tmp/cchLnLRU.s:32: Error: selected processor does not support `ldapr x0,[x0]'
```

Reply:
In the meantime i've bumped it to 8.3 this error may take some time to chase down. 3d2d06ba6